### DISCUSSION TITLE
MPD docker?

### DISCUSSION THREAD

**Discussion by Solarunit** [MusicPlayerDaemon/MPD#1794] [General]
I am new to mpd. I wonder is there a docker MPD image?

**Comment by MaxKellermann** [comment]
No

**Reply by Solarunit** [reply]
And not planning in the future? Was it discussed before?

**Reply by MaxKellermann** [reply]
I have never used Docker, I don't plan to ever use Docker, and that's all. If you find it useful, build one.

**Reply by Solarunit** [reply]
I see, I pretty new to docker, never build one myself.

**Reply by bonelifer** [reply]
@Solarunit here's one that was recently updated: https://hub.docker.com/r/tobi312/rpi-mpd/

**Reply by Solarunit** [reply]
Thanks, bookmarked!

**Reply by madickinson** [reply]
I learned a lot working on building mpd in a docker image and running it. By doing it yourself (building mpd in the image) you can be sure that it doesn't contain any shady or weird choices. The mpd software is well done and it makes sense to not have a docker image in addition that has to be maintained. As they say, "Don't half ass two things, whole ass one thing."
